### ISSUE TITLE
Improve report image appereance

### ISSUE THREAD

**Issue by luisgc93** [luisgc93/stock_reminder_bot#9]
The basic features of #7 are covered but the appearance of the report details needs some work.  

The current implementation for generating the image and adding the test uses the [Pillow library](https://pillow.readthedocs.io/en/stable/), which is not designed specifically for generating images based on a table.

An alternative approach would be to save the report data into a pandas data frame which would then be converted into a table and saved as a .png file.  I had to revert my attempt of this since the `dfi.export` method caused an `OSError error: Chrome executable not able to be found on your machine
`

https://github.com/luisgc93/stock_reminder_bot/commit/c5d88fcc7c979f5a70d4d0bd6bc84fabd49553c6

The approach would be something like this:
https://stackoverflow.com/questions/35634238/how-to-save-a-pandas-dataframe-table-as-a-png

I was able to create this locally but it raised an OSError when deployed:

![image](https://user-images.githubusercontent.com/32971373/104035897-ffb58400-51d2-11eb-9177-1d6cfa27ce05.png)

**Comment by luisgc93** [comment]
In order to make this work, try adding chrome through docker

**Comment by luisgc93** [comment]
Fixed:
https://twitter.com/stock_reminder/status/1350505139375894530?s=20

**Comment by Dheerkt** [comment]
Is alpine chrome what you ended up using to make this work? Facing a similar issue right now, and wondering how to solve it.

Really cool project btw!

**Comment by luisgc93** [comment]
> Is alpine chrome what you ended up using to make this work? Facing a similar issue right now with dataframe_image package, and wondering how to solve it.
> 
> Really cool project btw!

Hi! Glad you like the project!

I was able to save the image locally but setting up the chrome driver on heroku was an absolute nightmare. Instead, I ended up using this library:
https://pypi.org/project/dataframe-image/

If I remember correctly, what made this approach work was using `table_conversion=None` which bypasses the default `table_conversion='chrome'` and therefore doesn't need a chrome driver:
https://github.com/luisgc93/stock_reminder_bot/commit/ede93620e7ac940b7b4b5c8b00a25b113ad2e84d
